File: app/src/main/java/org/mifos/mobile/ui/views/AccountOverviewMvpView.java
package org.mifos.mobile.ui.views;

import org.mifos.mobile.ui.views.base.MVPView;

/**
 * @author Rajan Maurya
 *         On 16/10/17.
 */
public interface AccountOverviewMvpView extends MVPView {

    void showTotalLoanSavings(double totalLoan, double totalSavings);

    void showError(String message);
}

File: app/src/main/java/org/mifos/mobile/ui/views/AccountOverviewMvpView.kt
package org.mifos.mobile.ui.views

import org.mifos.mobile.ui.views.base.MVPView

/**
 * @author Rajan Maurya
 * On 16/10/17.
 */
interface AccountOverviewMvpView : MVPView {
    fun showTotalLoanSavings(totalLoan: Double?, totalSavings: Double?)
    fun showError(message: String?)
}